modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
> I agree with having that as a default and not having a flag, however this would still be a breaking change for libraries with extensions since you have to adapt the require statements?

Indeed ... and unfortunately extensions like frame_helper are required directly (there is no .rb file that does the loading, which we could adapt)... So back to your original idea :(